docs/developer-guide.md

```diff
@@ -223,3 +223,14 @@ You can run Shell Plus with the following command:
 Our project benefits from Sonarcloud integration, a static code analysis tool seamlessly integrated with Github. This integration is configured to automatically run on every pull request, identifying and reporting code issues for quick resolution. You can view the results and analysis insights on the [Sonarcloud dashboard](https://sonarcloud.io/project/overview?id=bcgov_cas-registration).
 
 For advanced customization and configuration, we've provided a `.sonarcloud.properties` file. This file allows you to fine-tune Sonarcloud analysis settings to suit your project's unique requirements and code quality standards.
+
+### Entity-Relationship Diagram (ERD) Generation
+
+We use [Mermaid](https://mermaid-js.github.io/mermaid/) to create ERD diagrams for our data models. The [django-diagram](https://github.com/nick-solly/django-diagram) package enables the generation of ERD diagrams specific to our Django models.
+
+To generate the most recent ERD diagram, navigate to the `bc_obps` directory (where the `manage.py` file is located) and execute the following command in your terminal:
+
+```bash
```

Comment:
This would be a nice-to-have, but could we add this to pre-commit or CI or somewhere to make sure we always remember to update the diagram if we change db stuff?

Reply:
You are very in sync with @andrea-williams :D 
I've created a tech debt for that: https://app.zenhub.com/workspaces/team-moose-sprint-board-60d216175a2689000ec774d0/issues/gh/bcgov/cas-registration/313